README.md

```diff
@@ -84,6 +84,8 @@ public void doSomething(DoSomethingCommand doSomethingCommand) {
     doSomethingUseCase.doSomething(doSomethingCommand);
 }
 ```
+Worker feature an adjustable retry count. They are set through the BPMN model in Camunda Platform 8. Camunda 7 worker have a 
```

Comment:
Suggestion:
Worker feature an adjustable retry count. In Camunda 8 IT can be set through the BPMN model on a specific service task. THE Camunda 7 worker have a default retry value of 3. The retry count can be configured as described in the [C7-specific README](...)